docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -62,12 +84,22 @@ def _import(mod, klass):
         return obj, ":obj:`{}.{{}}`".format(mod)
 
 
-def _section(header, mod_ext, other_lib, klass=None, exclude_mod=None):
+def _section(
+    header, mod_ext, other_lib, klass=None, exclude_mod=None, ufuncs=False
+):
     base_mod = "numpy" + mod_ext
     other_mod = other_lib + mod_ext
 
+    if ufuncs:
```

Comment:
why do you do this? `klass` and `exclude_mod` are `None` by default, so you just need to make sure you don't pass a non `None` value from the caller.

Reply:
I am joining it in case if someone pass not-None value from the caller. I can remove it, if you think id doesn't make sense

Reply:
let's not generalize things until it's really necessary. unnecessary generalization just makes the code hard to maintain for others.